docs/PG_CRON_SETUP_GUIDE.md

```
@@ -0,0 +1,252 @@
+#### 1. Install pg_cron Extension
+```sql
+-- Connect as PostgreSQL superuser
+CREATE EXTENSION IF NOT EXISTS pg_cron;
+
+-- Verify installation
+SELECT * FROM pg_extension WHERE extname = 'pg_cron';
+```
+
+#### 2. Configure PostgreSQL
+Add to `postgresql.conf`:
+```ini
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)